Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

```diff
@@ -111,4 +121,29 @@ private function getFunctionyTokens()
 
         return $tokens;
     }
+
+    /**
+     * Gets the name of tokens that are actually language construction.
+     *
+     * @staticvar int[] $tokens Token names.
```

Comment:
there is no such annotation in PSR5 proposal
https://github.com/php-fig/fig-standards/pull/169